pygotham/talks/models.py

```diff
@@ -98,3 +94,20 @@ def __str__(self):
     def is_accepted(self):
         """Return whether the instance is accepted."""
         return self.status == 'accepted'
+
+
+class CoPresenter(db.Model):
+
+    __tablename__ = 'copresenters'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    name = db.Column(db.String(255), info={'label': 'Name'}, nullable=False)
```

Comment:
The only columns in a join table should be its foreign keys. See [`roles_users`](https://github.com/PyGotham/pygotham/blob/ced260525b152bf8f2f5b4cef0831e900ddbecc3/pygotham/users/models.py#L11-L15) for an example.